astroquery/nist/tests/test_nist.py

```diff
@@ -45,6 +45,9 @@ def test_query_async(patch_get):
                                           linename="H I", get_query_payload=True)
     assert response['spectra'] == "H I"
     assert response['unit'] == nist.core.Nist.unit_code['nm']
+    response = nist.core.Nist.query_async(4000 * u.nm, 7000 * u.nm,
+                                          linename=["H I", "Fe I"], get_query_payload=True)
+    assert response["spectra"] == "H I Fe I"
```

Comment:
Based on https://physics.nist.gov/PhysRefData/ASD/Html/lineshelp.html#LINES_SPECTRA this should be
```suggestion
    assert response["spectra"] == "H I; Fe I"
```

Reply:
Applied.